mungegithub/mungers/release-note-label_test.go

```diff
@@ -38,107 +39,120 @@ var (
 func TestReleaseNoteLabel(t *testing.T) {
 	runtime.GOMAXPROCS(runtime.NumCPU())
 
+	var (
+		comment1 = "Release note process has not been followed"
+		comment2 = "The 'parent' PR of a cherry-pick PR must have one of the"
+	)
+
 	tests := []struct {
-		name        string
-		issue       *github.Issue
-		body        string
-		branch      string
-		secondIssue *github.Issue
-		mustHave    []string
-		mustNotHave []string
+		name             string
+		issue            *github.Issue
+		body             string
+		branch           string
+		secondIssue      *github.Issue
+		mustHave         []string
+		mustNotHave      []string
+		mustHaveComment1 bool
+		mustHaveComment2 bool
 	}{
 		{
 			name:        "LGTM with release-note",
 			issue:       github_test.Issue(botName, 1, []string{lgtmLabel, releaseNote}, true),
 			mustHave:    []string{lgtmLabel, releaseNote},
-			mustNotHave: []string{releaseNoteLabelNeeded},
+			mustNotHave: []string{releaseNoteLabelNeeded, doNotMergeLabel},
```

Comment:
Do we really need to test that it doesn't have `do-not-merge`? Issues could have `do-not-merge` for other reasons than release note issues.